Clarify that WV2 receives the same updates as Microsoft Edge (#1885)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/distribution.md

The Evergreen distribution mode ensures that your WebView2 app is taking advantage of the latest WebView2 features and security updates.  The Evergreen distribution mode has the following characteristics:

> *  The WebView2 Runtime updates automatically without more effort from you. It recevices the same Microsoft Edge updates described in [Release notes for Microsoft Edge Stable Channel](/deployedge/microsoft-edge-relnote-stable-channel) and [Release notes for Microsoft Edge Security Updates](/deployedge/microsoft-edge-relnotes-security).

Reviewer: ```suggestion
*  The WebView2 Runtime updates automatically without requiring any action from you. It receives the same Microsoft Edge updates that are described in [Release notes for Microsoft Edge Stable Channel](/deployedge/microsoft-edge-relnote-stable-channel) and [Release notes for Microsoft Edge Security Updates](/deployedge/microsoft-edge-relnotes-security).
```
